src/conventions/datapack_uninstallation.md

```diff
@@ -11,3 +11,105 @@ There are no rules enforcing how you should implement the uninstallation method,
+
+**Fun Fact :** If you are a beginner, you may not know that you can make mcfunction run slower tick cycles as you like.
+For example, run every 8 tick `schedule function <function> 8t` or every 1 second `schedule function <function> 20t or 1s` `t - tick`, `s - second`, `d - gameday`. Why am I talking about this? Because I've seen scoreboards used to do this before.
+
+> Pros
+> - Don't worry about the name 
+> - No need to disable datapack.
+>
+> Cons
+> - If not disabled, You must complete all schedule clear commands.
+> - There must be no reload after all schedule clear until the datapack file is removed.
+
+**3. Installation type**
+
+`.zip`
+> When uninstalling, if the user deletes the datapack file, they must exit game or stop server. when uninstalling 
+>
+> Pros
+> - Just drag it into the datapack folder. 
+
+`extract to folder`
+> No need to exit the game or stop the server. when uninstalling
+>
+> Cons
+> - You need to be sure that the user will extract the folder correctly. Including when you compress it into a .zip file, how do you make it?
+> - `<name>.zip/<name>/data`
+If it is like this, the user must click `Extract here`.
+> - `<name>.zip/data`
+If it is like this, the user must click `Extract to "<name>\"`
+
+**3. Various reload methods**
+
+ > 3.1 `/datapack disable <name>`
+ >
+ > 3.2 `/reload` (There is an issue with some server launchers.)
+ >
+ > 3.3 `Exit and Re-join game or Stop & Re-launch Server`
+
+**4. Conclusion from convention mod's opinion.**
+
+**Fact :** The `tick.json` file typically runs before `load.json`, sometimes resulting in an incomplete execute and installation. But we can set `load.json` for `setup` or `init` file to run before main or tick.
+```json
+{
+    "values": [
+        "convention:datapack/setup",
+        "convention:datapack/main"
+    ]
+}
+```
+
+If it was me, what method would I use? 
+> - Use `/schedule clear <function>` 
+> - Use `.zip` (Just drag & drop, good experience for the users)
+> 
+> Because it has pros is **"Don't worry about the name & No need to disable datapack"**  since `tick.json` is not used.
+And installing `.zip` will be able to force the user to *exit the game or stop the server (reload method 3.3)* to delete the datapack file. Because the datapack is not disabled.
+
+> - If schedule function command is more than 2
+> - Use `Use /datapack disable <static_name>` (convention.zip every update)
+> - Use `.zip` (Just drag & drop, good experience for the users)
+> - If you only want to use `datapack disable .zip`, you must tell the user not to extract the files.
+> 
+> Because we want the `setup` or `init` file to run first.
+Which we'll only use `datapack disable "file/convention.zip"` to have everything disabled in one command, with prohibits users from extracting files.
+And installing `.zip` will be able to force the user to *exit the game or stop the server (reload method 3.3)* to delete the datapack file. 
```

Comment:
```suggestion
> Because I want the `init` file to run first.
And installation as a `.zip` file will be able to force the user to *exit the game or stop the server (reload method 4.3)* to delete the datapack file. 
```